BESU-164: Add information about creating indexes for log queries. (#117)
hyperledger/besu-docs: docs/Reference/API-Methods.md

    ```bash tab="curl HTTP request"
    curl -X POST --data '{"jsonrpc":"2.0","method":"admin_changeLogLevel","params":["WARN"], "id":1}' http://127.0.0.1:8545
    ```

    ```bash tab="wscat WS request"
    {"jsonrpc":"2.0","method":"admin_changeLogLevel","params":["WARN"], "id":1}
    ```

    ```json tab="JSON result"
    {
     "jsonrpc": "2.0",
     "id": 1,
     "result": "Success"
    }
    ```

### admin_generateLogBloomCache

Generates cached log bloom indexes for blocks. APIs such as [`eth_getLogs`](#eth_getlogs)
and [`eth_getFilterLogs`](#eth_getfilterlogs) use the indexes for improved
performance.  

!!! note
    Each index file contains 100000 blocks. The last fragment of blocks less that 100000 are not indexed.

Must be manually executed to add new blocks to the indexes.

**Parameters**

`integer` - Block to start generating indexes

`integer` - Block to stop generating indexes

**Returns**

`result` : *object* - Log bloom index details:

> * `integer` : `startBlock` - Starting block for the generated indexes

Author: I'm unsure about the returns section, could you please confirm these?

Reviewer: ```suggestion
* `quantity` : `startBlock` - Starting block of the last requested cache generation
```

Author: Done

> * `integer` : `endBlock` - End block for the generated indexes

Reviewer: ```suggestion
* `quantity` : `endBlock` - Ending block of the last requested cache generation
```

> * `quantity` : `currentBlock` - Hexadecimal integer representing the latest

Reviewer: I think calling out the hexideicmal number is already taken care of by calling it a quantity.
```suggestion
* `quantity` : `currentBlock` - The most recent block that was added to the cache
```

   block indexed
* `boolean` : `indexing` - `true` if indexing is in progress

> * `boolean` : `requestAccepted` - `true` if the request to generate log bloom

Reviewer: ```suggestion
* `boolean` : `requestAccepted` - `true` if the request from this call to generate log bloom cache has been accepted
```

    }

    ```

### eth_getFilterLogs

Returns an array of [logs](../Concepts/Events-and-Logs.md) for the specified filter.

> Improve log retrieval performance by using [`admin_generateLogBloomCache`](admin_generatelogbloomcache).

Reviewer 2: Suggest rewording to make active: 

Use [`admin_generateLogBloomCache`](admin_generatelogbloomcache) to improve log retrieval performance.

### eth_getLogs

Returns an array of [logs](../Concepts/Events-and-Logs.md) matching a specified
filter object.

> Improve log retrieval performance by using [`admin_generateLogBloomCache`](admin_generatelogbloomcache).

Reviewer 2: Same comment as above about rewording to make active.
